concordia/admin/views.py

```diff
@@ -159,11 +160,16 @@ def admin_bulk_import_view(request):
                     continue
 
                 try:
+                    
+                    pattern = re.compile(SlugConverter.regex)
```

Comment:
This runs in a loop and I would recommend caching it either outside the loop or even at the module level since it can never change during the program's execution. Maybe hoist this up as `PROJECT_SLUG_RE` following PEP-8 for a module-level constant?